### PR TITLE
ITM-510

### PR DESCRIPTION
**Pull request by Ayal-Ciobotaru**
Added 'justification' as a requirement for `action` and added `medical_policies` controlled vocab

### CODE REVIEW

**Comment by Ayal-Ciobotaru** [comment]
Adjusting based on discussion with Darren

File: swagger/swagger.yaml [hunk 1]
```diff
@@ -1311,6 +1313,14 @@ components:
         - some
         - extensive
         - crowd
+    MedicalPoliciesEnum:
+      type: string
+      description: Directives issued by competent military authority
```
Comment (nextcen-dgemoets) [inline swagger/swagger.yaml:1318]: Could you change "Neutral" to "Treat All Neutrally"?

File: swagger/swagger.yaml [hunk 2]
```diff
@@ -609,8 +609,10 @@ components:
           type: string
           description: The political climate in a mission to inform decision-making
         medical_policies:
-          type: string
-          description: Medical policies in effect in a mission, to inform decision-making
+          type: array
+          description: A list of medical policies
```
Comment (nextcen-dgemoets) [inline swagger/swagger.yaml:613]: Could you add, "; omit this property if no special policy is in place."

File: swagger/swagger.yaml [hunk 3]
```diff
@@ -348,7 +348,7 @@ paths:
               schema:
                 $ref: "#/components/schemas/State"
         "400":
-          description: Invalid action or Session ID, or specified an intent action
+          description: Invalid action or Session ID, empty justification or specified an intent action
```
Comment (nextcen-dgemoets) [inline swagger/swagger.yaml:351]: Although this is true, we're not listing any other reasons the action can be invalid...so probably just leave it how it was before.

**Review by nextcen-dgemoets — COMMENTED** [review]
Obviously I'll test after the client PR is re-opened, but here are some preliminary comments.

File: swagger_server/itm/data/dryrun/test/sample.yaml [hunk 4]
```diff
@@ -19,7 +19,7 @@ state:
     communication_capability: both # current availability of internal and external communication, controlled vocab includes internal, external, both, neither
     roe: unused # rules of engagement to inform decision-making, but not to restrict decision space
     political_climate: unused # The political climate in a mission to inform decision-making
-    medical_policies: unused # Medical policies in effect in a mission, to inform decision-making
+    medical_policies: Treat All Neutrally # controlled vocab includes Treat All Neutrally, Treat Enemy LLE, Treat Civilian LLE, and Prioritize Mission
```
Comment (nextcen-dgemoets) [inline swagger_server/itm/data/dryrun/test/sample.yaml:22]: This string needs to be converted into an array with one string.

File: swagger_server/itm/data/metrics/test/sample.yaml [hunk 5]
```diff
@@ -19,7 +19,7 @@ state:
     communication_capability: both # current availability of internal and external communication, controlled vocab includes internal, external, both, neither
     roe: unused # rules of engagement to inform decision-making, but not to restrict decision space
     political_climate: unused # The political climate in a mission to inform decision-making
-    medical_policies: unused # Medical policies in effect in a mission, to inform decision-making
+    medical_policies: Treat All Neutrally # controlled vocab includes Treat All Neutrally, Treat Enemy LLE, Treat Civilian LLE, and Prioritize Mission
```
Comment (nextcen-dgemoets) [inline swagger_server/itm/data/metrics/test/sample.yaml:22]: This string needs to be converted into an array with one string.

**Review by nextcen-dgemoets — CHANGES_REQUESTED** [review]
Need to fix the sample; otherwise looks good.

**Review by nextcen-dgemoets — APPROVED** [review]

**Review by dereknop — APPROVED** [review]

**Pull request merged**